cpmpy/tools/xcsp3/README.md

```diff
@@ -0,0 +1,84 @@
+# CPMpy XCSP3 tools
+
+This directory contains a collection of tools for reading, loading and solving XCSP3 instances in CPMpy.
+
+What is included:
+- utilities for reading and loading XCSP3 instances
+- a PyTorch-compatible dataset class for loading (and downloading) XCSP3 instances
+- cli for solving individual XCSP3 instances and outputting the result in the competition format
```

Comment:
well, if we consider that part of this, then that is where the code should go, not in the init : )